src/app/modules/app/pages/users/components/user-form/user-form.component.ts

```diff
@@ -97,4 +106,21 @@ export class UserFormComponent implements OnInit {
       },
     });
   }
+
+  private updateUser(): void {
+    const user = this.userForm.value;
+    console.log(user);
+    const formattedBirthdate = formatDate(user.birthdate);
+    user.birthdate = formattedBirthdate;
+    delete user.country;
+    console.log(user);
+    this.userService.edit(user).subscribe({
+      next: user => {
+        this.dialogRef.close(user);
```

Comment:
Can we add Snackbar in case of success and error?

Reply:
https://github.com/JU-DEV-LatamBootcamp/Klockan-Frontend/assets/151553728/9aa6f3b8-d65f-4b1d-b2e6-d3c1611abbde

is already implemented